service/internal/auth/authn.go

```diff
@@ -528,7 +528,9 @@ func (a Authentication) isPublicRoute(path string) func(string) bool {
 			slog.Warn("error matching route", slog.String("route", route), slog.String("path", path), slog.String("error", err.Error()))
 			return false
 		}
-		slog.Debug("matching route", slog.String("route", route), slog.String("path", path), slog.Bool("matched", matched))
+		if matched {
+			slog.Debug("matching route", slog.String("route", route), slog.String("path", path), slog.Bool("matched", matched))
+		}
```

Comment:
Don't we want to capture both conditions here?  Maybe we need to introduce a trace level?

Reply:
If a route was not matched and not put through Casbin enforcement, I see the log as having little value. If we do want to capture it and move it below `debug` into a sort of `verbose-debug`, that would be fine with me. The trouble is there are a lot of value in debug logs, but it's difficult to sift through them to find the important ones with all the noise from these route path matching logs.